javascripts/application.js

```diff
@@ -407,6 +407,16 @@ const OC = {
                     });
                     return false;
                 }
+               
+                var maxLength = 128;
+      
+                uploadMedia.forEach(function (item, index) {
+                    var extension = item.file.name.split('.').pop();
+
+                    if (item.file.name.length > 128) {
+                    	  item.file.name = item.file.name.substring(0, maxLength - extension.length - 1) + '.' + extension;
```

Comment:
The file name is a readonly property.

```cpp
interface File : Blob {
  constructor(sequence<BlobPart> fileBits,
              USVString fileName,
              optional FilePropertyBag options = {});
  readonly attribute DOMString name;
  readonly attribute long long lastModified;
};
```

https://w3c.github.io/FileAPI/#file-section

This is causing [TypeError: setting getter-only property "name"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Getter_only)

Hier ist, was ein Dozent dazu geschrieben hat:

> gerade versuche ich, bei Stud.ip Videos mit Opencast hochzuladen. Wenn ich alle Angaben eingegeben habe und auch die Datei eingetragen gehe ich auf Hochladen. Dann erscheint sofort wieder die leere Maske und das Hochladen wird nicht vollzogen. Das habe ich jetzt mit zwei Browsern versucht (Safari, Firefox) mit dem selben Effekt.